### PR TITLE
Feature/Admin manage page

### PR DESCRIPTION
**Pull request by yunicho**
The admin page feature is completed
 - Both places and comment button to Read and Delete all in domain
 
Future css to be discussed

### CODE REVIEW

File: src/main/java/com/cracker/comment/service/CommentService.java [hunk 1]
```diff
@@ -119,4 +122,54 @@ public long updateByUser(Long commentId, CommentUpdateRequestDto commentUpdateRe
...
+        return dtos;
+    }
+
+    static class AdminCompareModifiedDesc implements Comparator<AdminCommentListRequestDto>{
+        @Override
+        public int compare(AdminCommentListRequestDto o1, AdminCommentListRequestDto o2){
+            return o2.getModifiedAt().compareTo(o1.getModifiedAt());
+        }
+    }
+//        List<AdminCommentListRequestDto> dtos = new ArrayList<AdminCommentListRequestDto>();
```
[the remark this answers is not in the document]
Comment (devpcjin) [reply src/main/java/com/cracker/comment/service/CommentService.java:151]: 주석 처리된 부분

File: src/main/java/com/cracker/place/controller/AdminController.java [hunk 2]
```diff
@@ -32,4 +38,19 @@ public PlaceDeleteResponseDto deletePlace(@PathVariable("id") Long id) {
 
         return placeDeleteResponseDto;
     }
+
+    @GetMapping("/admin/comments")
```
[the remark this answers is not in the document]
Comment (yunicho) [reply src/main/java/com/cracker/place/controller/AdminController.java:42]: 문제 있을까요? manage가 처음에 도메인 중복 나서 그랬는데 /admin/comments & /admin/places로 나누면서 필요없어졌어요

[the remark this answers is not in the document]
Comment (leverest96) [reply src/main/java/com/cracker/place/controller/AdminController.java:42]: manage 문제 없어요

File: src/main/java/com/cracker/comment/service/CommentService.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (yunicho) [reply src/main/java/com/cracker/comment/service/CommentService.java:151]: completed

File: src/main/java/com/cracker/place/controller/AdminController.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (devpcjin) [reply src/main/java/com/cracker/place/controller/AdminController.java:42]: 그냥 바뀌어서요 ㅋㅋ

**Pull request merged**